search/search.go

```diff
@@ -25,19 +26,13 @@ const (
 // Truncate lines to prevent sending over massive hunks, e.g. a minified file.
 // NOTE: We may end up truncating a valid flag key reference. We accept this risk
 //       and will handle hunks missing flag key references on the frontend.
-func truncateLine(line string) string {
-	// len(line) returns number of bytes, not num. characters, but it's a close enough
-	// approximation for our purposes
-	if len(line) <= maxLineCharCount {
+func truncateLine(line string, maxCharCount int) string {
```

Comment:
easier for testing, but also just a better design not to use a global constant within this function